assets/app/views/builds.html

```diff
@@ -17,72 +17,72 @@
           <div>
             
               <dt>Build strategy:</dt>
-              <dd>{{buildConfig.parameters.strategy.type}}</dd>
+              <dd>{{buildConfig.spec.strategy.type}}</dd>
 
           </div>
-          <div ng-switch="buildConfig.parameters.strategy.type">
-            <div ng-switch-when="STI">
-              <div ng-if="buildConfig.parameters.strategy.stiStrategy.image">
+          <div ng-switch="buildConfig.spec.strategy.type">
+            <div ng-switch-when="Source">
+              <div ng-if="buildConfig.spec.strategy.sourceStrategy.image">
 
                   <dt>Builder image:</dt>
-                  <dd>{{buildConfig.parameters.strategy.stiStrategy.image}}</dd>
+                  <dd>{{buildConfig.spec.strategy.sourceStrategy.image}}</dd>
```

Comment:
pretty sure this became `buildConfig.spec.strategy.sourceStrategy.from`, which we want to put through the filter